src/Hosting/Hosting/src/Internal/HostingApplication.cs

```diff
@@ -57,12 +56,153 @@ public void DisposeContext(Context context, Exception exception)
 
         public struct Context
         {
+            private LogContext _loggingContext;
+
             public HttpContext HttpContext { get; set; }
-            public IDisposable Scope { get; set; }
-            public long StartTimestamp { get; set; }
-            public bool EventLogEnabled { get; set; }
-            public Activity Activity { get; set; }
-            internal bool HasDiagnosticListener { get; set; }
+            public IDisposable Scope
+            {
+                // Get set value or return null
+                get => _loggingContext?.Scope;
```

Comment:
I was going to make this `readonly get` but the version of C# AspNetCore uses is too old. 

I think updating it will make Code check go crazy as every auto-property getter gets the `[Readonly]` attribute